storybook/.storybook/webpack.config.js

```diff
@@ -0,0 +1,10 @@
+var path = require('path');
+const storysource = (module.exports = function({ config }) {
```

Comment:
Why using a variable?

Reply:
Probably just a left over from some testing I did. Going to drop it.